picolibc.ld

```diff
@@ -159,7 +157,7 @@ SECTIONS
 	 * into the allocate ram addresses by the existing
 	 * data initialization code in crt0
 	 */
-	.tdata : @LDSCRIPT_ALIGN_WITH_INPUT@ {
+	.tdata : ALIGN(8) {
```

Comment:
This is not the same and I believe will break the overaligned TLS test when linking with ld.bfd.